target/linux/airoha/files/drivers/pinctrl/pinctrl-airoha.c

```diff
@@ -218,19 +218,49 @@
 #define GPIO17_FLASH_MODE_CFG			BIT(1)
 #define GPIO16_FLASH_MODE_CFG			BIT(0)
 
+/* System Setting Register 3 */
+#define REG_SSR3				0x00
+#define SSUSB_HSGMII_SEL			BIT(29)
+#define ETH_XSI_SEL_MASK			GENMASK(14, 13)
+#define ETH_XSI_XFI_USXGMII			(1 << 13)
+#define ETH_XSI_SGMII_HSGMII			(2 << 13)
```

Comment:
@merbanan please use BIT() and GENMASK() macro here

Reply:
Ack